public/scripts/play.js

```diff
@@ -161,8 +160,8 @@ function stopPlayer(e) {
   }
 }
 
-//Marianna
-//change animation based on the direction of the player
+// Marianna
+// change animation based on the direction of the player
 function changeAnimation(direction) {
```

Comment:
animation function

Reply:
This one just switches between the animations that are set for movement to the top, movement to the bottom and so on so the player always faces the right direction. For now those values are hardcoded but when we add player selection they will be dynamic.